nsfc-plugin/src/main/java/org/osc/controller/nsfc/exceptions/NullObjectReturnedNsfcException.java

```diff
@@ -0,0 +1,22 @@
+package org.osc.controller.nsfc.exceptions;
+
+import org.openstack4j.model.common.Resource;
+
+public class NullObjectReturnedNsfcException extends NsfcException {
```

Comment:
not sure if this exception is needed.

The API calls if successful would not return nulls...

Reply:
Sometimes that is what happens.
https://github.com/dmitryintel/sdn-controller-nsfc-plugin/blob/196d30b775b0f84a0670de58722f4b4298b7fc15/nsfc-plugin/src/test/java/org/osc/controller/nsfc/LiveEnvIntegrationTest.java#L359